common/timer_migrate.go

```diff
@@ -4,14 +4,15 @@ import (
 	"fmt"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/lavanet/lava/common/types"
```

Comment:
why do we have changes in this file?

Reply:
anyhow, that should go away once you rebase/merge from main.